src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -151,21 +152,46 @@ protected override void StopProcessing()
 #region "Internals"
 
         /// <summary>
-        /// Run a command.
+        /// Run shutdown command
         /// </summary>
-        protected void RunCommand(String command, String args) {
+        protected void RunShutdown(String args) {
+            string shutdownPath = "/sbin/shutdown";
+            const string altShutdownPath = "/usr/sbin/shutdown";
+
+            if (!File.Exists(shutdownPath) && File.Exists(altShutdownPath))
+            {
+                shutdownPath = altShutdownPath;
```

Comment:
Make sense to save to static cache after first discover?

Reply:
yes, let me figure out why it's not working in CI first